apps/admin-server/src/middleware.ts

```diff
@@ -9,6 +9,8 @@ export default async function middleware(req: NextRequest) {
   // ignore
   if (req.nextUrl.pathname.startsWith('/_next')) return res; // internal urls
   if (req.nextUrl.pathname.startsWith('/favicon') ) return res;
+  if (req.nextUrl.pathname.startsWith('/health')) return res;
```

Comment:
Just to check. Do you still need this? Because as I understand it, it rewrites /health to /api/health

Reply:
Yes, you do, otherwise the rewrite doesn't happen